.pre-commit-config.yaml

```diff
@@ -33,22 +33,8 @@ repos:
           - "--line-length"
           - "119"
         exclude: ^django_fsm_log/migrations|^docs/
-  - repo: https://github.com/pycqa/isort
-    rev: 5.12.0
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    # Ruff version.
+    rev: 'v0.0.257'
```

Comment:
Just from the order, https://github.com/charliermarsh/ruff-pre-commit says it should be run before other things like black

Reply:
You can also remove yesqa

Reply:
thanks

Reply:
I also removed pyupgrade as it is also covered by ruff